test/test_kvtree_bcast.c

```diff
@@ -0,0 +1,66 @@
+#include "test_kvtree.h"
+#include "test_kvtree_util.h"
+#include "mpi.h"
+#include "stdio.h"
+
+
+int main(int argc, char** argv){
+  int rc = TEST_PASS;
+  int rank, ranks, kvtree_rc;
+  int val_of_one, val_of_two, val_of_three;
+  const char one[3] = "ONE", two[3] = "TWO", three[5] = "THREE";
```

Comment:
I think we are missing room for the trailing NUL here, in which case, the string might have some garbage characters after it.  Probably need to use either ```const char one[4] = "ONE"``` or ```const char one[] = "ONE"```